server/api/db/sqldb/db.py

```diff
@@ -991,6 +991,15 @@ def tag_artifacts(
         project: str,
     ):
         artifacts_keys = [artifact.key for artifact in artifacts]
+        if not artifacts_keys:
+            logger.debug(
+                "No artifacts to tag",
+                project=project,
+                tag=tag_name,
+                artifacts=artifacts,
+            )
+            return
```

Comment:
can that be in a separate pr?

Reply:
You got it https://github.com/mlrun/mlrun/pull/6066